gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,687 @@
+id: gigamon
+metric_id: gigamon
+backend_only: false
+facets:
+  - groups:
+      - Web Access
+    name: URL Path
+    path: http.url_details.path
+    source: log
+  - groups:
+      - DNS
+    name: DNS ID
+    path: dns.id
+    source: log
+  - groups:
+      - DNS
+    name: Response Code
+    path: dns.flags.rcode
+    source: log
+  - groups:
+      - Web Access
+    name: Referer
+    path: http.referer
+    source: log
+  - groups:
+      - Web Access
+    name: Request ID
+    path: http.request_id
+    source: log
+  - groups:
+      - Web Access
+    name: URL Host
+    path: http.url_details.host
+    source: log
+  - groups:
+      - Web Access
+    name: OS
+    path: http.useragent_details.os.family
+    source: log
+  - facetType: list
+    groups:
+      - gigamon
+    name: Gigamon APP ID
```

Comment:
May have gotten missed in the back-and-forth, but I mentioned it's the `path` that needs namespacing with `gigamon.` prefix and not the name. The paths need to be unique across all integrations, and that's why we ask contributors to namespace the custom facets. 

This one should look like: 
```
  - facetType: list
    groups:
      - gigamon
    name: App ID
    path: gigamon.app_id
    source: log
    type: string
```

With changing the facet, you'll also need a remapper to map `app_id` to `gigamon.app_id`: 
```
    - type: attribute-remapper
      name: Map 'app_id' to 'gigamon.app_id'
      enabled: true
      sources:
        - app_id
      sourceType: attribute
      target: gigamon.app_id
      targetType: attribute
      preserveSource: false
      overrideOnConflict: false
```

These remappers would be needed for each custom facet that we add the `gigamon.` prefix to

Reply:
@vishalshah-dd , should we do the below facet type maunally in the yaml file ? 
  - facetType: list
    groups:
      - gigamon
    name: App ID
    path: gigamon.app_id
    source: log
    type: string

Reply:
Thanks @emarsha94 , Will work on it and will update it.

Reply:
@emarsha94 , could you please review https://github.com/DataDog/integrations-extras/pull/2356/#issuecomment-2313238119 this thread and confirm ? Thanks.